java-squid/src/test/java/org/sonar/java/cfg/CFGTest.java

```diff
@@ -964,76 +902,23 @@ public void prefix_operators() {
 
   @Test
   public void try_statement() {
-    CFG cfg = buildCFG("void fun() {try {System.out.println('');} finally { System.out.println(''); }}");
```

Comment:
This seems to be an error during your rebasing on master as those tests the new implementation of try/finally. Please keep this test correct.